data/json/items/gunmod/crossbow_reload_mechanisms.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": "cord_crossbow_mech",
+    "type": "GUNMOD",
+    "name": "cord crossbow reload mechanism",
+    "description": "A small clip and a long string designed to attach to the back of a crossbow, letting you draw the string back more easily.  Takes a bit longer to use though.",
+    "weight": 20,
```

Comment:
Weight and volume for those gunmods are probably better replaced with proper units, like grams or mililiters or whatever fits.